### PR TITLE
Windoor Collision Fix

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Fixes bug with windoor collision.

## Why It's Good For The Game

Fixes bug with windoor collision.

Fixes https://github.com/tgstation/TerraGov-Marine-Corps/issues/5575

## Changelog
:cl:
fix: Fixes bug with windoor collision.
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/doors/windowdoor.dm [hunk 1]
```diff
@@ -86,6 +86,9 @@
 	. = ..()
 	if(istype(mover) && CHECK_BITFIELD(mover.flags_pass, PASSGLASS))
 		return TRUE
+	if(get_dir(loc, target) == dir) //Make sure looking at appropriate border
+		return
```
Comment (Hughgent) [inline code/game/objects/machinery/doors/windowdoor.dm:90]:
```suggestion
		return FALSE
```

Comment (Hughgent) [reply code/game/objects/machinery/doors/windowdoor.dm:90]: This seems to be the common return for "cannot" pass through.

**Comment by Wayland-Smithy** [comment]
Latest commit appears to make windoors one-way, only passable if you are on their tile.
blF9E7IBCA

**Comment by Surrealaser** [comment]
> Latest commit appears to make windoors one-way, only passable if you are on their tile.

Had to revert Hugh's suggestion; it works now.

File: code/game/objects/machinery/doors/windowdoor.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/machinery/doors/windowdoor.dm:90]: This suggestion is wrong

Comment (TiviPlus) [inline code/game/objects/machinery/doors/windowdoor.dm:90]:
```suggestion
	if(istype(mover) && CHECK_BITFIELD(mover.flags_pass, PASSGLASS))
		return TRUE
	if(get_dir(loc, target) == dir) //Make sure looking at appropriate border
		return ..()
```

File: code/game/objects/machinery/doors/windowdoor.dm [hunk 2]
```diff
@@ -86,6 +86,9 @@
 	. = ..()
```
Comment (TiviPlus) [inline code/game/objects/machinery/doors/windowdoor.dm:86]: Duplicate supercal
```suggestion
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**